requirements/functests.in

```diff
@@ -1,6 +1,6 @@
 pip-tools
 pip-sync-faster
-httpretty
+responses
```

Comment:
This requirement originally comes from the cookiecutters project, so I guess we make the change there?